ONSdigital/eq-survey-runner
# Eq model validation

**What**
Initial validation set up. n.b exceptions and full documentation to follow

**How to test**
1. Check the code and design patterns
2. Run the test_integer.py

**Who can test**
Anyone but @LJBabbage
